.github/workflows/testpypi-publish.yml

```diff
@@ -0,0 +1,30 @@
+name: TestPyPI Publish
+
+on:
+  push:
+    branches:
+      - contrib
+
+jobs:
+  test_pypi_publish:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.x
+
+      - name: Install dependencies
+        run: pip install --upgrade pip twine setuptools wheel
+
+      - name: Build and Publish to TestPyPI
+        env:
+          TWINE_USERNAME: __token__
+          TWINE_PASSWORD: ${{ secrets.TEST_PYPI_API_TOKEN }}
+        run: |
+          python setup.py sdist bdist_wheel
+          twine upload --repository-url https://test.pypi.org/legacy/ dist/*
```

Comment:
remove file?

Reply:
This particular was used during testing and it uploaded the package to TestPyPi, which is like PyPi but for testing before actual deployment

Reply:
Right, I get that. 
I assumed, based on the "contrib" branch filtering in this config, you were using this to try out and test  the distribution tools and process without worrying about affecting the real index. Is that still the case? if so why do you want to merge this into main branch?
Or do you want to keep publishing to TestPyPi instance on all future pushes?

Reply:
Oh, I get it now. We don't have to merge it to the branch. It has been removed now. Thanks.